README.md

```diff
@@ -116,17 +120,16 @@ Once Heroku has updated you can [create a new game](https://play.battlesnake.com
 
 **At this point you should feel comfortable making changes to your code and deploying those changes to Heroku!**
 
-
-
 ## Developing Your Battlesnake Further
 
 Now you have everything you need to start making your Battlesnake super smart! Here are a few more helpful tips:
 
-* Keeping your logs open in a second window (using `heroku logs --tail`) is helpful for watching server activity and debugging any problems with your Battlesnake.
+- Keeping your logs open in a second window (using `heroku logs --tail`) is helpful for watching server activity and debugging any problems with your Battlesnake.
+
+- You can use the Javascript [console.log](https://nodejs.org/api/console.html) to output information to your server logs. This is very useful for debugging logic in your code during Battlesnake games.
 
-* You can use the Javascript [console.log](https://nodejs.org/api/console.html) to output information to your server logs. This is very useful for debugging logic in your code during Battlesnake games.
+- Review the [Battlesnake API Docs](https://docs.battlesnake.com/snake-api) to learn what information is provided with each command. You can also output the data to your logs:
```

Comment:
This sentence ends on a colon, I think?

Reply:
It does, I think because of the example directly below it